module/vision/VisualMesh/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 # Add the VisualMesh project to be built
 add_subdirectory(VisualMesh)
 
+add_compile_definitions(CL_TARGET_OPENCL_VERSION=120)
```

Comment:
The new version of the visual mesh should fix up a lot of the warnings from that code

Reply:
Is that on a branch I can merge into this branch?

Reply:
No somebody has to go and update the visual mesh code to work with the new architecture which is still somewhat in progress